app/assets/javascripts/admin/addon/components/admin-config-area-cards/about/general-settings.gjs

```diff
@@ -41,12 +80,20 @@ export default class AdminConfigAreasAboutGeneralSettings extends Component {
       <p class="admin-config-area-card__additional-help">
         {{i18n "admin.config_areas.about.banner_image_help"}}
       </p>
-      <UppyImageUploader />
+      <UppyImageUploader
+        @type="site_setting"
+        @imageUrl={{this.aboutBannerImage}}
+      />
     </div>
     <DButton
       @label="admin.config_areas.about.update"
       @action={{this.save}}
-      class="btn-primary"
+      class="btn-primary save-card"
```

Comment:
Let's make this more specific for config areas, should be `admin-config-area-card__btn-save`

Reply:
Also we should disable all save buttons while the AJAX call is happening